### PR TITLE
scalafix 0.10.0

### PR DESCRIPTION
**Pull request by bjaglin**
See https://github.com/scalacenter/scalafix/releases/tag/v0.10.0.

### CODE REVIEW

**Comment by bjaglin** [comment]
@joan38 any chance you can approve the worflow to run so I get feedback? Thanks in advance!

**Comment by joan38** [comment]
Done

**Comment by bjaglin** [comment]
Thanks, it looks good. Once the final release is cut, it would be nice to get a major release of your client ASAP.

That's because of the breaking change explained in https://github.com/scalacenter/scalafix/releases/tag/v0.10.0-RC1: if rules start building against scalafix 0.10.x, they will unfortunately fail with existing clients using scalafix 0.9.x (https://github.com/scalacenter/scalafix/pull/1562 tried to improve the error message, but it's not trivial). OrganizeImports being the most important one, let's wait until a 0.10.x-built mill-scalafix release is published until a 0.10.x-built OrganizeImports is published, as suggested in https://github.com/liancheng/scalafix-organize-imports/pull/264.

**Comment by joan38** [comment]
Thanks @bjaglin for taking care of this!

**Comment by bjaglin** [comment]
@joan38 0.10.0 is out

**Comment by rvacaru** [comment]
Hi @joan38, I'm Razvan, [GSoC contributor for scalafix](https://summerofcode.withgoogle.com/programs/2022/projects/gQ08FcXb) and @bjaglin is my GSoC mentor.

TLDR: We'd like a release of mill-scalafix which includes this PR.

As part of the project I'm cross compiling scalafix in scala 3, however in [scalafix-core there's a macro](https://github.com/scalacenter/scalafix/pull/1562/files#diff-e007588dec3c4ce6cc43c872df6bbd1cb8e6e352d889c07ce6bb83e7d88a5a9b) for better error message / readability.
We want to remove this hack for the next scalafix-core/scalafix release, and by having a release of mill-scalafix depending on scalafix 0.10.x we'd still have good error readability on such release. 

Would this be possible?

**Comment by joan38** [comment]
Hi @rvacaru !
Oh I did not realize that was not released yet.
I will merge the open PRs and make a release now.

**Comment by joan38** [comment]
Also note that we have snapshot releases on every commit, the latest being `0.2.8-6-dd9c632-SNAPSHOT`

**Comment by joan38** [comment]
`0.2.9` is out

**Comment by rvacaru** [comment]
Thanks a lot for the quick response @joan38! :)
